### ISSUE TITLE
callback url-hez user assurance

### ISSUE THREAD

**Issue by magwas** [edemo/Joomla_oauth_plugin#24]

**Comment by utopszkij** [comment]
a joomla seo szisztémával ezt nem tudom összehozni.
<callbackurl>?proxiuserid=<proxyuserid> kellene. Továbbá itt nem csak megnézhet hanem modosithat is.

**Comment by krosza** [comment]
App specifikus tanúsítás folyamata
1. személy és lakcímkártya egyeztetés
2. app email cím alapján app reg adatok lekérése
3. app reg adatok és lakcímkártya egyeztetés ( pl név  tényleg jó-e)
4.ADA email elkérése, ADA tanúsítói felületen tanúsítás indítása ( ez az email nem feltétlen kell, hogy egyezzen az app email-el)
5. Tanúsítás ( titkos kód alapján) és ha ok,  (magyar és) app specifikus igazolás kiadása
6. ellenőrzés: app  reg adatok újra lekérése vagy user belép az app-ba és megnézi, hogy megvan-e a hozzáférése, amihez az igazolás kellett.
Megjegyzés:előfordulhat, bár nem valószínű, hogy név azonosság(pl. Kovács)  esetén (és feltéve, hogy az app és az ADA email különböző és az app nem az emailt használja a belépéshez azonosítóként) az ügyfél az app  esetében egy másik  Kovács  email címét adja meg és  így a tanúsítás után hiába frissítjük az app oldalát  a user-nek nincs app specifikus igazolása mert azt egy másik user ADA ID -hez kapcsolódik ( akinek az email címét a user megadta) . Ez az eset azonban csak akkor lehetséges , ha az app enged  nem ADA-s reg-et ezért ráérünk foglalkozni az esettel, ha lesz ilyen app, akinek az ADA elött sok usere regisztrált korábban más módon.

**Comment by utopszkij** [comment]
A 2.pont: app specifikus prixiuserid alapján app reg adatok ellenörzése. Ehhez viszont az kell, hogy az asurel "manager" jogú regisztrát legyen az applikációban! (hiszen az app reg adatok nem publikusak minden asurer számára!

**Comment by krosza** [comment]
Igen, de azt az app eldontheti hogy ez  konkretan milyen jogositvanyokat jelent, es csak az app altal meghatalmazott tanusitok kapjak ezt.

**Comment by utopszkij** [comment]
OK.
